howto/HACKME.md

```diff
@@ -148,3 +152,51 @@ dist/Debug/GNU-Linux/amos-ss17-proj4 can0
 ```
 
 To modify the build system best open the project in Netbeans (see above).
+
+## Set-Up on Virtual Machine
+If the simulator should be installed on a virtual linux machine just do the following steps to get the simulator running.
+
+ISO-TP
+
+As like in the Set-Up for the Raspberry PI, ISO-TP should be installed. If the CAN connection should be started while your system is starting, do the following steps.
+
+1. Load the can modules into the `modules.conf` file. This file can be found in your systems root directory via this path `/etc/modules-load.d`
+
+2. Open the file with the command `sudo nano modules.conf`
+
+3. Insert 
+```
+vcan
+isotp
+```
+Leave the nano terminal with `ctrl + x`, save with `j` and enter. Now open the file and check if the file was modified.
```

Comment:
I think it's a bit too verbose to describe here how to use nano. Especially since everyone has his preferred editor.
I guess it's enough to say what to write into which file.

Reply:
I will shorten that than.